app/dashboard/dashboard.component.e2e-spec.ts

```diff
@@ -0,0 +1,22 @@
+import { browser, by, element } from 'protractor';
+
+describe('Dashboard', () => {
+
+  beforeEach(() => {
+    browser.get('/dashboard');
+  });
+
+  it('should show dashboard text', () => {
+
+    let dashboardCardElement = element(by.css('hip-dashboard md-card'));
+
+    browser.wait(function () {
+      return dashboardCardElement.isPresent();
+    }, 60000);
+
+    expect(dashboardCardElement.getText()).toEqual('Dashboard');
+
+  });
+
+
+});
```

Comment:
__[tslint]__
* file should end with a newline

<!-- Comment by [SideCI](https://sideci.com) -->